src/docs/guides/27-how-to-handle-provider-onboarding-and-identity-verification/index.md

```diff
@@ -0,0 +1,230 @@
+[StripePayoutPage](https://github.com/sharetribe/ftw-daily/tree/master/src/containers/StripePayoutPage),
+[StripeConnectAccountForm](https://github.com/sharetribe/ftw-daily/tree/master/src/forms/StripeConnectAccountForm),
+and
+[stripeConnectAccount.duck.js](https://github.com/sharetribe/ftw-daily/tree/master/src/ducks/stripeConnectAccount.duck.js)
+where the Connect onboarding is mostly handled. You can also check out
+[PR #1234](https://github.com/sharetribe/ftw-daily/pull/1234) where you
+can find all the code changes we did to default FTW.
+
+### 2. Creating provider Stripe Account
+
+With Stripe Connect Onboarding, you need to collect minimum information
+from your providers when creating a
+[Stripe Account](https://www.sharetribe.com/api-reference/marketplace.html#create-stripe-account)
+for them. It's recommended for you to pass in for the
+`create-stripe-account` call the following information:
+
+- `country`: A mandatory field determining the country of residence for
+  the provider.
+- `bankAccountToken`: Stripe
+  [bank account token](https://stripe.com/docs/api#create_bank_account_token)
+  for the user. Recommended to pass in at this phase to minimize the
+  steps in the onboarding process.
+- `requestedCapabilities`: Required capabilities for payments to work in
+  Flex are `card_payments` and `transfers`. You can read more about
+  capabilities from
+  [How Capabilities work?](/background/how-capabilities-work/)
+
+![Creating Stripe account](stripePayoutForm.png)
+
+**Note:** On the `EditListingWizard`, the modal with
+`StripeConnectAccountForm` is shown if the user doesn't have a Stripe
+account yet or if there is some information missing from the account.
+The modal will be shown only if the user is publishing the listing. This
+means that users can update already published listing even if their
+Stripe account is in the restricted state but the can't publish new
+listings.
+
+### 3. Fetching the information about Stripe account
+
+If the user already has the Stripe account you need to fetch the
+up-to-date account data from through Flex API. This way we can warn the
+users if there is some required information missing from their Stripe
+account.
+
+The account data is returned after each create and update Stripe account
+API calls so there is no need for separate API call in these cases.
+
+In FTW the Stripe account is fetched in `loadData` function on
+`StripePayoutPage` and on `EditListingPage`.
+
+### 4. Creating Stripe Account Link
+
+Stripe Account Links are a mechanism for enabling your providers to
+access Stripe Connect Onboarding UI. You need to
+[create an account link](https://www.sharetribe.com/api-reference/marketplace.html#create-stripe-account-link)
+and provide the return URLs for success and failure cases. After
+creating an account link the user will be redirecet to Connect
+onboarding.
+
+![Connect onboarding](connectOnboarding.png)
+
+If there are any requirements missing from the Stripe account we will
```

Comment:
There should be something about how this is controlled, i.e. the `type` param described.

Reply:
@sktoiva I'm not sure if I understood your comment correctly but I added a paragraph about checking the requirement to "Fetching information about s Stripe Account" section https://github.com/sharetribe/flex-docs/pull/221/files#diff-6656356dc74fe1301153668e960c9aadR97